vast/vastutil/testdata/unwrap/unwrapper_multiads_3.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<VAST version="2.0"/>
```

Comment:
Please add an empty line for the end of file. (all these xml files)

Reply:
Thank you for your notice. I find out there is an empty line in all our current .json and .xml files. I will check add add for them.